src/extension.ts

```diff
@@ -127,15 +127,15 @@ export function activate(context: vscode.ExtensionContext): void {
 
     context.subscriptions.push(vscode.commands.registerCommand(
         `${SETTINGS_VIEW_NAME}.toggleScreencast`,
-        (target?: CDPTarget) => {
+        (target?: CDPTarget, isJsDebugProxiedCDPConnection = false) => {
```

Comment:
Is there a purpose of defaulting `isJsDebugProxiedCDPConnection` to false here if we already do it in `ScreencastPanel.createOrShow`?

Reply:
Not particularly. We could just make this an optional param. No strong preference on my part.